source/MaterialXCore/Document.cpp

```diff
@@ -357,11 +359,42 @@ void convertParameterToInput(DocumentPtr doc)
                 {
                     // TODO: Determine based on usage whether to make
                     // the input a uniform. 
-                    newInput->setIsUniform(false);
+                    newInput->setIsUniform(true);
                 }
+                anyConverted = true;
+            }
+        }
+    }
+    return anyConverted;
+}
+
+bool Document::convertUniformInputsToParameters()
```

Comment:
I'm probably just missing it, but it this being used somewhere other than the test?

Reply:
Yes. It's being used in test. And can be used for Arnold rendering when checked-in.